### PR TITLE
Added Playbooks

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
Documentation restructuring 
 - links pointing to fortinet-fortisoar repo
 - Restructured as per agreed upon template
 - Added description to contents
 - Adjusted heading levels
 - renamed image filenames to lowercase separated by '-' from camel case
 
 Mantis#0828086 
 - Validated that Alerts/Incidents linking to the corresponding techniques or sub-technique records

### CODE REVIEW

File: playbooks/globalVariables.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (cs-suyog-jinturkar) [inline playbooks/globalVariables.json:1]: Why this change?

**Comment by cs-suyog-jinturkar** [comment]
I don't see this "renamed image filenames to lowercase separated by '-' from camel case" in PR, am I missing anything here?

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: playbooks/globalVariables.json (hunk 1 above)
Comment (cs-rohit-khune) [reply playbooks/globalVariables.json:1]: @cs-suyog-jinturkar `globalVariables.json` is the default file that contains global variables and gets created while playbooks export. However, in these playbooks, there is no global variable used, so this file comes empty.

**Review by cs-suyog-jinturkar — APPROVED** [review]
